tests/test_htsget_server.py

```diff
@@ -36,7 +36,7 @@ def get_headers(username=USERNAME, password=PASSWORD):
 
 
 def test_remove_objects():
-    cohorts = ["test-htsget", "1000Genomes"]
+    cohorts = ["test-htsget", "1000genomes"]
```

Comment:
There's a `get_ingest_file` in this file that has 1000Genomes uppercased -- unsure if that one needs to change too.